contracts/Root.sol

```diff
@@ -79,10 +81,16 @@ contract Root is Ownable {
     }
 
     function getAddress(bytes name, bytes proof) internal view returns (address) {
+
+        // Add "nic." to the front of the name.
```

Comment:
Wouldn't it make more sense to do this in the  same place we add `_ens.`?

Reply:
Not really, due to the fact that we are using a shared library.